docs/sources/features/datasources/mssql.md

```diff
@@ -226,7 +226,7 @@ WHERE
 ORDER BY 1
 ```
 
-When above query are used in a graph panel the result will be two series named `Metric A` and `Metric B` with value of `valueOne` and `valueTwo` plotted over `time`.
+When above query is used in a graph panel the result will be two series named `Metric A` and `Metric B` with value of `valueOne` and `valueTwo` plotted over `time`.
```

Comment:
Change sentence to:
When the above query is used in a graph panel, it will produce two series named `Metric A` and `Metric B` with the values `valueOne` and `valueTwo` plotted over `time`.

Thanks!

Reply:
I have made the changes as per the suggestion.